pkg/registry/types.go

```diff
@@ -14,6 +14,24 @@ var (
 	ErrPackageNotInDatabase = errors.New("Package not in database")
 )
 
+// BundleImageAlreadyAddedErr is an error that describes a bundle is already added
```

Comment:
How about adding an `ExitCoder` interface to simplify/allow extension of `opm` error code usage: 
```go
type ExitCoder interface {
	ExitCode() int
}

type BundleImageAlreadyAddedErr struct {
	ErrorString string
}

...

func (e BundleImageAlreadyAddedErr) ExitCode() int {
	return 2
}
```

Reply:
Disregard this comment, the exit code is bringing CLI problems into a non-CLI-specific library.